### DISCUSSION TITLE
How to use AR.js in wechat miniprogram ?

### DISCUSSION THREAD

**Discussion by pxim** [AR-js-org/AR.js#247] [Q&A]
**Do you want to request a *feature* or report a *bug*?**
How to use AR.js in wechat miniprogram ?

Example :
This is wechat  miniprogram code
![gh_347f68ff6d49_258](https://user-images.githubusercontent.com/5111701/112568150-fd031c00-8e1c-11eb-8c80-73a127e7a72b.jpg)

This is pattern
![xiaoxiongmao](https://user-images.githubusercontent.com/5111701/112569870-ee6a3400-8e1f-11eb-98d0-9013a3bf11da.png)

Open wechat, scan the small program code, and then click the recognition experience to scan the pattern
This is the effect
![f0b860694f33bbe44e1efb5ac189176](https://user-images.githubusercontent.com/5111701/112569420-16a56300-8e1f-11eb-80fb-3628641f97e3.jpg)

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**Please mention other relevant information such as the browser version, Operating System and Device Name**

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Comment by magixmin** [comment]
////////////////////////////////////////////////////////////////////////////////
        //          handle arToolkitSource
        ////////////////////////////////////////////////////////////////////////////////

        var arToolkitSource = new THREEx.ArToolkitSource({
            sourceType : 'webcam',
            sourceWidth: 480,
            sourceHeight: 640,
        })

        arToolkitSource.init(function onReady(){
            // use a resize to fullscreen mobile devices
            setTimeout(function() {
                onResize()
            }, 1000);
        })

        // handle resize
        window.addEventListener('resize', function(){
            onResize()
        })

        // listener for end loading of NFT marker
        window.addEventListener('arjs-nft-loaded', function(ev){
          console.log(ev);
        })

 I `m do AR. in wechat H5 . but. [
![891672296893_ pic](https://user-images.githubusercontent.com/68672136/209915099-d18f8e1a-5a5b-4f59-a9c6-a1b8a6b9b6c9.jpg)
](url)

There is now image show. on it .

**Reply by kalwalt** [reply]
Have you some errors in the log console?
